test/test_lava_test_plans.py

```diff
@@ -0,0 +1,4 @@
+import lava_test_plans
+
+def test_bla():
+    pass
```

Comment:
you might want to replace this with a real test.

Reply:
added some